src/category_theory/limits/opposites.lean

```diff
@@ -0,0 +1,122 @@
+-- Copyright (c) 2019 Scott Morrison. All rights reserved.
+-- Released under Apache 2.0 license as described in the file LICENSE.
+-- Authors: Scott Morrison
+
+import category_theory.limits.limits
+
+universes v u
+
+open category_theory
+open category_theory.functor
+
+namespace category_theory.limits
+
+variables {C : Sort u} [𝒞 : category.{v+1} C]
+include 𝒞
+variables {J : Type v} [small_category J]
+
+variables {F : J ⥤ Cᵒᵖ}
+
+def cone_of_cocone_left_op (c : cocone F.left_op) : cone F :=
+{ X := op c.X,
+  π := nat_trans.right_op (c.ι ≫ (const.op_obj_unop (op c.X)).hom), }
+
+@[simp] lemma cone_of_cocone_left_op_X (c : cocone F.left_op) :
+  (cone_of_cocone_left_op c).X = op c.X :=
+rfl
+@[simp] lemma cone_of_cocone_left_op_π_app (c : cocone F.left_op) (j) :
+  (cone_of_cocone_left_op c).π.app j = (c.ι.app (op j)).op :=
+begin
```

Comment:
Do we want to turn this into 1-liners?
`by {dsimp [cone_of_cocone_left_op], simp}`?

Reply:
Sure. Done.